src/Makefile.am

```diff
@@ -471,6 +471,10 @@ if HARDEN
 	$(AM_V_at) READELF=$(READELF) OBJDUMP=$(OBJDUMP) $(top_srcdir)/contrib/devtools/security-check.py < $(bin_PROGRAMS)
 endif
 
+check-formatting:
+	@echo "Checking source formatting style..."
+	(cd $(top_srcdir)/src; ../contrib/devtools/clang-format.py check clang-format-3.8 `cat ./.formatted-files`)
```

Comment:
clang-format-3.8 is not included into Ubuntu trusty, so I wonder how our Travis could use it since is based on that Ubuntu version and I don't see any change in the `.travis.yml` to add clang-format-3.8 from an external source.